usermanagerlib/src/test/java/org/pesmypetcare/usermanagerlib/clients/MedicationManagerClientTest.java

```diff
@@ -0,0 +1,152 @@
+package org.pesmypetcare.usermanagerlib.clients;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.pesmypetcare.usermanagerlib.datacontainers.DateTime;
+import org.pesmypetcare.usermanagerlib.datacontainers.Medication;
+import org.pesmypetcare.usermanagerlib.datacontainers.MedicationData;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.BDDMockito.given;
+
+public class MedicationManagerClientTest {
+    private static final String ACCESS_TOKEN = "my-token";
+    private static final StringBuilder STATUS_OK = new StringBuilder("200");
+    private static final int STATUS_OK_INT = 200;
+    private static String SHOULD_BE_200 = "Should return response code 200";
+    private static String SHOULD_RETURN_MED_LIST = "Should return medication data list";
+
+    private String owner;
+    private String petName;
+    private String name;
+    private MedicationData medicationData;
+    private Medication medication;
+    private List<Medication> medicationList;
+    private DateTime date;
+    private DateTime date2;
+    private String field;
+    private Double value;
+    private StringBuilder jsonMedicationData;
+    private StringBuilder jsonAllMedications;
+
+    @Mock
+    private TaskManager taskManager;
+
+    @InjectMocks
+    private MedicationManagerClient client = new MedicationManagerClient();
+
+    @Rule
+    public MockitoRule mockitoRule = MockitoJUnit.rule();
+
+    @Before
+    public void setUp() {
+        owner = "Manolo";
+        petName = "Kawaguchi";
+        name = "Ibuprofeno";
+        date = new DateTime("2020-02-13T10:30:00");
+        date2 = new DateTime("2021-02-13T10:30:00");
+        field = "quantity";
+        value = 60.0;
+        medicationData = new MedicationData(0.0, 2, 3);
+        medication = new Medication(date.toString(), name, medicationData);
+        medicationList = new ArrayList<>();
+        medicationList.add(medication);
+        jsonMedicationData = new StringBuilder("{\n"
+                + "      \"quantity\": \"0.0\",\n"
+                + "      \"duration\": \"2\",\n"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "      \"duration\": \"2\",\n" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5403182)